### ISSUE TITLE
tagbar cause window flash with NERD_tree

### ISSUE THREAD

**Issue by jackywu** [majutsushi/tagbar#364]
In Vim 8.0, open NERD_tree window at left split and Tagbar at right split, and the main area have several split and vsplit windows，and then move cursor to a file in NERD_tree window，press 'go' to preview file content, these windows will flash abnormally. This problem only appear with tagbar opened. After close the tagbar split windows，everything is ok. 

I have screenshot below.

the abnormal behavior：
![2016-11-22 14 30 42](https://cloud.githubusercontent.com/assets/587955/20513568/ada9f40e-b0c1-11e6-96ad-912410edf78e.gif)

the normal behavior：
![2016-11-22 14 41 05](https://cloud.githubusercontent.com/assets/587955/20513595/eb579554-b0c1-11e6-8bf5-608b35d335c5.gif)

Vim: "VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Sep 14 2016 03:42:03)",

**Comment by majutsushi** [comment]
I am not able to reproduce this, even by using your own configuration, and the GIFs are a bit hard to follow, so I'm not sure I understand the issue properly. Maybe this is Mac-specific, or it has been fixed in a newer Vim version?

**Comment by alerque** [comment]
Is it possible this is related to the issue mentioned in #532? Did you ever work out what was going on here @jackywu?

I can't replicate this either at this point so I'm not sure what to do next. If this is still an issue for anyone of course we can look into it, but if no comment in the next week or so I'll probably close this until we hear otherwise.
